src/index.ts

```diff
@@ -161,38 +154,20 @@ export class AzureDataStore {
     });
   }
 
-  /**
-   * Read content from azure blob storage.
-   * @param key
-   * @param callback
-   */
-//   public get (key: any, callback: any): void {
-//     this.opts.blob.getBlobToText(this.container, this.getFullKey(key), { disableContentMD5Validation: true}, (err, text, result, response) => {
-//       if (err && response.statusCode === 404)
-//       {
-//         return callback(Errors.notFoundError(err));
-//       }
-//       else if (response.statusCode !== 200 && err)
-//       {
-//         return callback(err);
-//       }
-//       return callback(null, Buffer.from(text, 'utf-8'));
-//     });
-// }
-
   /**
    * Check for the existence of the given key.
    * @param key
    * @param callback
    */
   public has (key: any, callback: any): void {
+    callback = once(callback);
     this.opts.blob.doesBlobExist(this.container, this.getFullKey(key), (err, result, response) => {
       if (err) {
-        callback(err, false);
+        return callback(err, false);
```

Comment:
For my understanding: why add `return`?

Reply:
it wont have any effect here, will clear the callback statements